{"repository":"https://github.com/ghostsquad/swarmci","dockerfilePath":"Dockerfile","startTime":1672206770888,"clone":{"stdout":"","stderr":"Cloning into '/tmp/dinghy-analysis/ghostsquad/swarmci'...\n","error":null,"commit":"d26587cabfcee6d084789383775d0775279a89fb"},"originalBuild":{"startTimestamp":1672206771745,"endTimestamp":1672206793307,"error":{"code":1,"killed":false,"signal":null,"cmd":"docker build --no-cache --force-rm -t swarmci:latest -f /tmp/dinghy-analysis/ghostsquad/swarmci/Dockerfile ."},"stdout":"Sending build context to Docker daemon   59.9kB\r\r\nStep 1/7 : FROM python:3-alpine\n3-alpine: Pulling from library/python\nc158987b0551: Already exists\n7ec3a91b66d9: Already exists\nc89fd98887b3: Pulling fs layer\n877ccf16716c: Pulling fs layer\n3cfcd1e4ebae: Pulling fs layer\n877ccf16716c: Verifying Checksum\n877ccf16716c: Download complete\n3cfcd1e4ebae: Verifying Checksum\n3cfcd1e4ebae: Download complete\nc89fd98887b3: Verifying Checksum\nc89fd98887b3: Download complete\nc89fd98887b3: Pull complete\n877ccf16716c: Pull complete\n3cfcd1e4ebae: Pull complete\nDigest: sha256:af8fef83397b3886ed93d2c81bf3b4e70d39c0789c1c6feb1ecb86ca9bc42a0a\nStatus: Downloaded newer image for python:3-alpine\n ---> 18ccdc39ea28\nStep 2/7 : RUN mkdir -p /usr/src/app\n ---> Running in c28df2b1f155\nRemoving intermediate container c28df2b1f155\n ---> a1fbf6ce3beb\nStep 3/7 : WORKDIR /usr/src/app\n ---> Running in b301b0d6fef4\nRemoving intermediate container b301b0d6fef4\n ---> 77f6cd6b740b\nStep 4/7 : COPY setup.py /usr/src/app\n ---> bdf1a6bc9bde\nStep 5/7 : RUN pip install -e .\n ---> Running in 55da4d4f72d3\nObtaining file:///usr/src/app\n  Preparing metadata (setup.py): started\n  Preparing metadata (setup.py): finished with status 'error'\n\u001b[91m  error: subprocess-exited-with-error\n  \n  × python setup.py egg_info did not run successfully.\n  │ exit code: 1\n  ╰─> [8 lines of output]\n      Traceback (most recent call last):\n        File \"<string>\", line 2, in <module>\n        File \"<pip-setuptools-caller>\", line 34, in <module>\n        File \"/usr/src/app/setup.py\", line 18, in <module>\n          with open('swarmci/version.py') as f:\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^\n        File \"<frozen codecs>\", line 906, in open\n      FileNotFoundError: [Errno 2] No such file or directory: 'swarmci/version.py'\n      [end of output]\n  \n  note: This error originates from a subprocess, and is likely not a problem with pip.\n\u001b[0m\u001b[91merror: metadata-generation-failed\n\n× Encountered error while generating package metadata.\n╰─> See above for output.\n\nnote: This is an issue with the package mentioned above, not pip.\nhint: See above for details.\n\u001b[0mRemoving intermediate container 55da4d4f72d3\n","stderr":"The command '/bin/sh -c pip install -e .' returned a non-zero code: 1\n"},"endTime":1672206793371}